shared/constants/permissions.ts

```diff
@@ -18,20 +22,52 @@ export const RestrictedMethods = Object.freeze({
 } as const);
 
 ///: BEGIN:ONLY_INCLUDE_IN(flask)
+
 export const PermissionNamespaces = Object.freeze({
   wallet_snap_: 'wallet_snap_*',
 } as const);
 
-export const EndowmentPermissions = Object.freeze({
+const StableEndowmentPermissions = Object.freeze({
   'endowment:network-access': 'endowment:network-access',
-  'endowment:long-running': 'endowment:long-running',
   'endowment:transaction-insight': 'endowment:transaction-insight',
   'endowment:cronjob': 'endowment:cronjob',
   'endowment:ethereum-provider': 'endowment:ethereum-provider',
   'endowment:rpc': 'endowment:rpc',
 } as const);
 
+const FlaskEndowmentPermissions = Object.freeze({
+  ...StableEndowmentPermissions,
+  'endowment:long-running': 'endowment:long-running',
+} as const);
+
 // Methods / permissions in external packages that we are temporarily excluding.
-export const ExcludedSnapPermissions = new Set([]);
-export const ExcludedSnapEndowments = new Set(['endowment:keyring']);
+const ExcludedFlaskSnapPermissions = {
```

Comment:
If this is final source of truth for this and similar specifications, it would probably be useful if it's frozen with `Object.freeze`. That would prevent any potential accidental change of it somewhere else.